(GH-878) Fixing install with packages.config - should print out the packages to install

andmos opened the pull request:
Trying a fix for #878.

Added a check for ``*.config`` files and added logging of the packages involved.
Also logs that we actually install from a ``*.config`` file. 

Closes #878

ferventcoder commented:
Hello @andmos - thanks for your contribution! 

Please ensure you've read over CONTRIBUTING.md and please fix up your commit messages as directed in that document. 

We'll be looking for the two commits to squash to just one commit. The doc also mentions how to push the changes back up to this PR. Thanks!

ferventcoder commented:
Also I noticed GitHub doesn't know how to associate your GH account with your commits. If that is not intentional, you may wish to set your username and user email in your git config :)

Hammerstad commented:
It looks like he has his work mail in his config.

He will need to [add that email address to his github account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/).

gep13 commented on Scenarios.md:
```diff
@@ -362,6 +362,8 @@
  * should not have inconclusive package result
  * should not have warning package result
+ * should print out package from config file in message
+ * should specify config file is beeing used in message
```
@andmos slight typo here.  Should this be `is being used`

gep13 commented on src/chocolatey.tests.integration/scenarios/InstallScenarios.cs:
```diff
@@ -435,6 +435,30 @@ public void should_not_have_warning_package_result()
             }
+
+            [Fact]
+            public void should_specify_config_file_is_beeing_used_in_message()
```
@andmos same typo here...

andmos commented:
There, I have squashed the commits and added a more explanatory text to the commit message.
Also changed the e-mail in the ``.gitconfig`` file, don't know if it helped?

ferventcoder commented:
> Also changed the e-mail in the .gitconfig file, don't know if it helped?

![image](https://cloud.githubusercontent.com/assets/63502/17223298/906d3a4c-54c1-11e6-9259-cc60c6fe104b.png)

andmos commented:
Should be good now!

bpjohannessen commented:
Is this being looked at? I'm about to start writing PowerShell code that solves this issue in my installer script..

ferventcoder commented:
@bpjohannessen Yes. This is planned to go into 0.9.10.4 (vNext).

ferventcoder commented:
@andmos There is only one bit of missing behavior. When the packages.config is not the only package name being passed.

ferventcoder commented:
Merged into stable at https://github.com/chocolatey/choco/commit/66b0107d3faeb40bc13c98a7c70ac7250ea0b65c
